queries/postgres_manager.py

```diff
@@ -0,0 +1,40 @@
+import psycopg2
+from os import environ
+from pydantic import BaseModel
+
+
+class DataBase(BaseModel):
+    dbname: str = environ['DB_NAME']
+    user: str = environ['USER_NM']
+    password: str = environ['PASSWORD']
+    host: str = environ['HOST']
+
+pgdb = DataBase()
+
+def start_connection():
+    conn_to_db = psycopg2.connect(dbname=pgdb.dbname,
+                             user=pgdb.user,
+                             password=pgdb.password,
+                             host=pgdb.host
+                            )
+    return conn_to_db
+
+def close_connection(data_base):
+        data_base.close()
+
+# TODO: isolation levels
+def run_postgres_query(
+        postgres_query,
+        values=[]

```

Comment:
never use mutable object as default value in python 
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments